gr-utils/python/modtool/cli/add.py

```diff
@@ -122,6 +124,7 @@ def get_arglist(self):
                                             default='',
                                             show_default=False)
 
+
```

Comment:
Looks like extra whitespace added here

Reply:
@mormj Yes, I'll correct the whitespaces.